apps/files/src/views/Sidebar.vue

```diff
@@ -189,7 +203,8 @@ export default {
 		 * @return {string}
 		 */
 		subtitle() {
-			return `${this.size}, ${this.time}`
+			const starredIndicator = this.fileInfo.isFavourited ? '★ ' : ''
```

Comment:
```suggestion
			const starredIndicator = this.fileInfo.isFavourited ? '⭐ ' : ''
```

If we want to make it more discoverable / prominent that the file is a favorite we might want to use this instead.

Reply:
Is this accessible? The grey one would meet the contrast requirements I would hope.

CC @JuliaKirschenheuter

Reply:
I think we would have to use an icon with alt text / aria-label, right @JuliaKirschenheuter @marcoambrosini?

Reply:
Well we have `--icon-starred-yellow` as an icon that fulfills accessibility requirements for bright and dark mode.

Reply:
The problem is that the NcAppSidebar only accepts a string in that subline. I suggest we add the text starred beside the star, so a screen reader would read out the info too. 
Contrast is ok, yes.

Reply:
@marcoambrosini would that work space-wise and with translations? Otherwise it might be better to have the indicator as an icon in the top right of the file preview/icon as it was before, just not as big (and with an outline). Then it matches how it looks like in the file list.

Reply:
I will add the text only in the aria label and not in the paragraph. We can't add just an icon to the sidebar preview. It's either the toggle button or nothing.

Reply:
Also not possible, there it's either the switch or nothing. I suggest we go ahead with this and we modify the sidebar component so that we can slot in a proper icon there as @susnux suggests

Reply:
sorry for a stupid question, but why we can't just use an normal material icon with `aria-label` in that place?

Reply:
> Also not possible, there it's either the switch or nothing.

Why is this not possible? Since the current solution is not accessible ("star" is not understandable as "favorite", and we can not add an aria-label) we need to do it otherwise.

Can we please do:
- Just use an icon on the top right of the filetype icon, non-interactive like in the file list
- The action to toggle it would be in the action menu, like you did here

@marcoambrosini @szaimen

Reply:
> Just use an icon on the top right of the filetype icon, non-interactive like in the file list

![Screenshot from 2023-11-22 13-44-55](https://github.com/nextcloud/server/assets/6078378/7c501302-a50c-4ab8-9221-e629ed7662e1)

and we have already everything we need

Reply:
> Since the current solution is not accessible ("star" is not understandable as "favorite", and we can not add an aria-label) we need to do it otherwise.

I don't think that this is not accessible.

Emoji is read by a screen reader. It could be unclear that "star" means "favorite". But it is the same for all users. Any user needs to understand that "star" is "a starred file" which means "favorite".

This would be good to have a description here. But not required if we appeal to a common user experience of "starring".

Reply:
Ok, thanks for the assessment @ShGKme – then @AndyScherzinger and I would agree we should merge this as-is for now.

In the long term however it could possibly be better to have an image with aria-label which is more explicit, which we could also place on the top right of the filetype icon so it is symmetric to the files list.